src/main/java/emissary/core/IBaseDataObjectXmlCodecs.java

```diff
@@ -432,6 +433,7 @@ public ElementEncoders(
      */
     public static final ElementDecoder DEFAULT_STRING_OBJECT_DECODER = (elements, ibdo, ibdoMethodName) -> {
         final Method method = getIbdoMethod(ibdoMethodName, String.class, Object.class);
+        final LinkedListMultimap<String, Object> parameters = LinkedListMultimap.create(100);
```

Comment:
Another approach is to update `IbdoMethodNames`to  `public static final String SET_PARAMETER = "appendParameter";` and make the following updates. 
```suggestion
final Method method = getIbdoMethod(ibdoMethodName, String.class, CharSequence.class);
```